tests/testvectors/updateelectoratetx_tests.json

```diff
@@ -0,0 +1,134 @@
+[
+  {
+    "transaction": {
+      "kind": "string:bns/create_proposal",
+      "chainId": "string:iov-exchangenet",
+      "title": "string:Hello 0.303118290291134",
+      "description": "string:Hello 0.23847208324147773",
+      "author": "string:tiov1c9eprq0gxdmwl9u25j568zj7ylqgc7ajyu8wxr",
+      "electionRuleId": 2,
+      "action": {
```

Comment:
@jleni , for context, what's happening with this data is that the [CreateProposalMsg](https://github.com/iov-one/weave/blob/master/x/gov/codec.proto#L189) is wrapping the [UpdateElectorateMsg](https://github.com/iov-one/weave/blob/master/x/gov/codec.proto#L255).

I'm not sure how to handle the translation from `action` to [raw_option](https://github.com/iov-one/weave/blob/master/x/gov/codec.proto#L194).  Do you want the `action` test data to be protobuf encoded and use the property name `raw_option` so that it can simply be copied or can/should the Ledger app itself do the encoding?

Reply:
The `action` must be displayed in the Ledger, i.e. the Ledger app needs to be able to parse the content of `raw_option` (which is one of the binary fields inside of `bytes`).  So a structured form like this makes sense to me.

Reply:
No problem. 
We typically decode the bytes into internal types in the device, which then are formatted, paged, etc. and shown in the screen.
The json fields are used by the unit tests to generate independently (other separate piece of code) an expected output.
Device and expected outputs are then compared.
As long as we can understand the matching between the json and the proto fields, is all ok. It does not matter if the json shows nested or flatten fields.

Reply:
Thanks, guys.